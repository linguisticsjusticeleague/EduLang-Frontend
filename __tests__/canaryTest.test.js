import React from "react";
import { render } from "@testing-library/react-native";
import Checking from "../Test";

test("canary test", () => {
  const sum = 2 + 2;
  expect(sum).toBe(4);
});
test("Checks react native testing library is working", () => {
  const helloWorldText = "Hello World!";

  const { toJSON, getByText } = render(<Checking />);

  const foundHelloWorldText = getByText(helloWorldText);

  expect(foundHelloWorldText.props.children).toEqual(helloWorldText);
  expect(toJSON()).toMatchSnapshot();
});
