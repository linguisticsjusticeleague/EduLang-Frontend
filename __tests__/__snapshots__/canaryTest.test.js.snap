// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checks react native is testing library is working 1`] = `
<View>
  <Text>
    Hello World!
  </Text>
</View>
`;

exports[`Checks react native testing library is working 1`] = `
<View>
  <Text>
    Hello World!
  </Text>
</View>
`;
